modules/serverless-rn-1-23-0.adoc

```diff
@@ -24,12 +24,7 @@
 
 * The {FunctionsProductName} Technology Preview now uses Source-to-Image (S2I) by default to build container images.
 
-////
-not identified yet
-
-[id="fixed-issues-1-23-0_{context}"]
-== Fixed issues
-////
+* The Kafka channel implementation was updated to include some improvements to the config map and data plane. If you are using the previous Kafka channel implementation, this is migrated to the new implementation automatically.
```

Comment:
In SRVKE-1251 we're discussing the options for documenting this with more details and there is a proposed text, feel free to chime in that issue.